Letterbook.Api.Tests/Resources/WebFinger/Examples.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System.Net;
+using Letterbook.Api.Tests.Support;
+using Letterbook.Api.Tests.Support.Extensions;
+using Letterbook.Core;
+using Letterbook.Core.Models.WebFinger;
+using Xunit.Abstractions;
+
+namespace Letterbook.Api.Tests.Resources.WebFinger;
+
+/*
+
+    https://docs.joinmastodon.org/spec/webfinger/
+
+    WebFinger as described in RFC 7033 is a spec that defines a method for resolving links to a resource, given only a URI on a particular server. 
+    
+    This allows anyone to look up where a resource is located without having to know its exact location beforehand; for example, by email or phone number. 
+    
+    This lookup is directed at the endpoint /.well-known/webfinger, and a resource query parameter is passed along with the lookup. 
+    
+    The resource URI used with Mastodon is the acct: URI as described in RFC 7565, with the username of a profile that is hosted on a particular domain.
+
+ */
+
+public class Examples : IDisposable
```

Comment:
Can you just name this WebFingerTests?

```suggestion
public class WebFingerTests: IDisposable
```

Reply:
Latest has this change 👍